benchmarking/repo_driver.py

```diff
@@ -37,6 +37,7 @@
 getParser().add_argument("--commit_file",
     help="The file saves the last commit hash that the regression has finished. " +
     "If this argument is specified and is valid, the --commit has no use.")
+getParser().add_argument("--env", help="environment variables passed to runtime binary")
```

Comment:
I suspect that if repo_driver doesn't need to do anything with this argument, it can be passed right through.